search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -145,11 +144,7 @@ class TemporalEntityAttributeService(
             WHERE temporal_entity_attribute.entity_id = :entity_id
             """.trimIndent()
 
-        val expandedAttrsList =
-            attrs.map {
-                expandJsonLdKey(it, contextLink)!!
```

Comment:
the `contextLink` param of this method should no longer be used.

Reply:
of course, strange that my IDE does not notify that